index.bs

```diff
@@ -1411,14 +1426,14 @@ partial interface MLGraphBuilder {
     </div>
 </div>
 
-### The concat() method ### {#api-mlgraphbuilder-concat}
+### The {{MLGraphBuilder/concat()}} method ### {#api-mlgraphbuilder-concat}
 Concatenates the input tensors along a given axis.
 <script type=idl>
 partial interface MLGraphBuilder {
   MLOperand concat(sequence<MLOperand> inputs, unsigned long axis);
 };
 </script>
-<div algorithm=concat>
+<div class="note">
```

Comment:
Unclear why this is now a *note* class element as opposed to an algorithm identified div section.

Reply:
It's a descriptive section and it is not algorithm. 
If it's not a note, then its plain text, but definitely not an algorithm.

Reply:
I can remove the note-ness and leave this as plain text describing the method arguments. It is not part of an algorithm.

If that is fine, then I can do this change for all the other places as well.